lib/ship_quiet_logistics/commands/processor.rb

```diff
@@ -11,20 +11,28 @@ def initialize(blackboard, queue)
 
         @config = ShipQuietLogistics.configuration
       end
-
       def call
         messages_count.times do
           document = next_document
           next if !document || document.nil?
+          case document_type document.namespace
+            when 'RMADocument'
+              config.process_rma_handler.(document)
```

Comment:
Yes @hugobast, I added it though what i was saying is that it is not yet used as it would be used in the future for RMA processing, which we don't do at this time.